tests/test_schema_validate.py

```
@@ -0,0 +1,219 @@
+        ) as f:
+            f.write(
+                '<?xml version="1.0" encoding="UTF-8"?>\n'
+                "<MzIdentML>\n"
+                "  <UnclosedTag>\n"  # Missing closing tag
+                "</MzIdentML>\n"
+            )
+            temp_file = f.name
+
+        try:
+            # Malformed XML should raise an exception during parsing
+            with pytest.raises(Exception):
```

Ruff (0.14.2)

Do not assert blind exception: `Exception`

(B017)